source/agora/consensus/data/Enrollment.d

```diff
@@ -41,6 +41,9 @@ public struct Enrollment
     /// X: random seed, The nth image of random value
     public Hash random_seed;
 
+    /// the block height where the random seed belongs to
+    public ulong random_seed_height;
```

Comment:
I will need a bit more time to think about a way to handle this, so I don't think we can get it merged this sprint, sorry.